lib/v2/hashnode/blog.js

```diff
@@ -0,0 +1,62 @@
+const got = require('@/utils/got');
+const { art } = require('@/utils/render');
+const path = require('path');
+const timezone = require('@/utils/timezone');
+
+const baseApiUrl = 'https://api.hashnode.com';
+
+module.exports = async (ctx) => {
+    const username = ctx.params.username;
+    if (!username) {
+        return;
+    }
+
+    const query = `
+    {
+        user(username: "${username}") {
+            publication {
+                posts{
+                    slug
+                    title
+                    brief
+                    coverImage
+                    dateAdded
+                }
+            }
+        }
+    }
+    `;
+
+    const userUrl = `https://${username}.hashnode.dev`;
+    const response = await got({
+        method: 'POST',
+        url: baseApiUrl,
+        headers: {
+            Referer: userUrl,
+            'Content-type': 'application/json',
+        },
+        body: JSON.stringify({ query }),
+    });
+
+    const publication = response.data.data.user.publication;
+    if (!publication) {
+        return;
+    }
+
+    const list = publication.posts;
+    ctx.state.data = {
+        title: `Hashnode by ${username}`,
+        link: userUrl,
+        item: list
+            .map((item) => ({
+                title: item.title,
+                description: art(path.join(__dirname, 'templates/description.art'), {
+                    image: item.coverImage,
+                    brief: item.brief,
+                }),
+                pubDate: timezone(item.dateAdded, +8),
```

Comment:
API data of `item.dateAdded` has an example value of `2022-06-19T08:11:11.103Z` which does not require timezone shifting.
```suggestion
                pubDate: parseDate(item.dateAdded),
```
docs https://docs.rsshub.app/joinus/pub-date.html

Reply:
Ok, I have finished the modification.